rdt/hyper_transformer.py

```diff
@@ -288,7 +288,7 @@ def get_final_output_columns(self, field):
             else:
                 final_outputs.append(output)
 
-        return final_outputs
+        return sorted(final_outputs, reverse=True)
```

Comment:
This method returns values like `[column.is_null, column.value]`, but since we expect the column order to be the other way around (i.e. `[column.value, column.is_null]`) I'm reversing the order.